File: ios/SwiftKotlination/SwiftKotlinationUITests/Sources/Network/Mocks/Response.swift
struct Response: Codable {
    var file: File?
    var error: NetworkError?
    var dataTask: URLSessionDataTaskMock
    
    init(_ file: File? = nil, error: NetworkError? = nil, dataTask: URLSessionDataTaskMock = URLSessionDataTaskMock()) {
        self.file = file
        self.error = error
        self.dataTask = dataTask
    }
}

File: android/SwiftKotlination/app/src/test/kotlin/fr/jhandguy/swiftkotlination/network/Response.kt
package fr.jhandguy.swiftkotlination.network

import fr.jhandguy.swiftkotlination.network.mocks.HttpURLConnectionMock

class Response(file: File? = null, error: NetworkError? = null) {
    val urlConnection: HttpURLConnectionMock = HttpURLConnectionMock(file, error)
}